vertx-mail-client/src/main/java/io/vertx/ext/mail/package-info.java

```diff
@@ -91,10 +80,8 @@
  *
  * == Mail-client data objects
  *
- * The objects used by the MailClient service are all @DataObject classes, which means that they have a few properties
- * and can be converted from and to JSON. In Java the regular getter and setter methods can be used for the properties
- * in Groovy you can use the property names directly. In JavaScript and Ruby, you can use a JSON object directly to
- * construct or read the properties. The objects can be used fluently. 
+ * The objects used by the MailClient are all @DataObject classes, which means that they have a few properties
```

Comment:
You should word this so it makes sense in all languages.

E.g. something like "Mails are sent using JSON, here's an example of sending a mail", then show an example.

The example will be translated into each language too so the Java user will see a DataObject being sent, but the JS user will see some JSON being sent.

Don't mention things like DataObject as that won't mean anything to a JS/Ruby user.

Does that make sense? So always think how the comment will appear to different languages - they always must be language neutral.